src/app/tabs/runTab/settings.js

```diff
@@ -98,7 +98,7 @@ class SettingsUI {
         </label>
         <div class="${css.account}">
           <select data-id="runTabSelectAccount" name="txorigin" class="form-control ${css.select} custom-select" id="txorigin"></select>
-          ${copyToClipboard(() => document.querySelector('#runTabView #txorigin').value)}
+          <div style="margin-left: 0px;">${copyToClipboard(() => document.querySelector('#runTabView #txorigin').value)}</div>
```

Comment:
Its probably a bad idea to put in the inline style.  But the css definition .account_3NyvTG i {
    margin-left: 12px;
}
is winning anyway.

Reply:
it's a hack, this is a wraper for copy to clipboard which we want to do only here and we have to do this since we 'generate' it.